### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by tardigrader** [tardigrader/cv-page#1]
Conform to WCAG 2.0

See: http://wave.webaim.org/report#/https://zork.nu/CV

- html lang=sv missing
- Anchor elements without content, ie icons are missing text content. 
- Aria
